doc/swish/erlang.tex

```diff
@@ -2686,3 +2782,37 @@ \subsection{Data-Encoding Utilities}
 For \code{base64url-encode-bytevector} it includes \code{-} and \code{\_}.
 In keeping with Section 3.1 of~\cite{rfc4648}, these procedures
 do not introduce line breaks in the output.
+
+% ----------------------------------------------------------------------------
+\subsection{Macro Utilities}
+
+% ----------------------------------------------------------------------------
+\defineentry{pretty-syntax-violation}
+\begin{procedure}
+  \code{(pretty-syntax-violation \var{msg} \var{form} \opt{\var{subform} \opt{\var{who}}})}
+\end{procedure}
+\returns{} never
+
+The \code{pretty-syntax-violation} procedure raises a syntax violation.
+It differs from the native \code{syntax-violation} in that it formats
+\var{form} and \var{subform} using \code{pretty-format} abbreviations
+and it does not attempt to infer a who condition when \var{who} is not
+provided as this can produce confusing results in error messages
+involving match patterns.
+To provide more readable exception messages, it constructs the formatted
+message condition by calling \code{pretty-print} before raising the exception
```

Comment:
add a comma after "raising the exception"

Reply:
Fixed.